Assets/Plugins/Source/Editor/EditorWindows/EOSSettingsWindow.cs

```diff
@@ -92,10 +93,47 @@ public static void ShowWindow()
             window.SetIsEmbedded(false);
         }
 
+        protected override void OnEnable()
+        {
+            base.OnEnable();
+            ProductConfig.PlatformConfigsDeploymentUpdatedEvent += ReloadDeploymentSettingsForPlatformConfigEditors;
+        }
+
+        protected override void OnDestroy()
+        {
+            ProductConfig.PlatformConfigsDeploymentUpdatedEvent -= ReloadDeploymentSettingsForPlatformConfigEditors;
+            base.OnDestroy();
+        }
+
+        private void ReloadDeploymentSettingsForPlatformConfigEditors(object sender, ProductConfig.PlatformConfigsDeploymentUpdatedEventArgs e)
+        {
+            // For each of the platform config editors
+            foreach (IPlatformConfigEditor platformConfigEditor in _platformConfigEditors)
```

Comment:
Is it ever possible for `_platformConfigEditors` to become added to as a result of running this method? I'm guessing by this point, all platform config editors are already subscribed.

Reply:
No - it is not possible for `_platformConfigEditors` to be added as a result of this function that particular field member is never altered - it is only enumerated.